tools/devtool

```diff
@@ -1286,6 +1287,9 @@ cmd_test() {
     # dir. Let's fix that.
     cmd_fix_perms
 
+    # Log CPU information
+    say "$(lscpu)"
```

Comment:
I did do this when I was first testing but it is a bit noisy in a harness that is already quite busy which is why I added it after the tests had run in the event that debug information is needed. Even if the test fails it will still output the CPU information. Here is a failing testing with the proposed logging,
https://buildkite.com/firecracker/mattschlebusch-perf-baselines/builds/17#0183d07e-184f-4ea8-8e70-bf97d87a72c2

Reply:
I think the only piece of info from `lscpu` that we are interested in is `Model name`. We could just print that before the kernel version.